README.md

```diff
@@ -30,13 +30,14 @@ Each pattern creates the following infrastructure on the VPC:
 
 - The virtual server (VSI) pattern deploys identical virtual servers across the VSI subnet tier in each VPC
 - The Red Hat OpenShift Kubernetes (ROKS) pattern deploys identical clusters across the VSI subnet tier in each VPC
+- The VPC pattern deploys networking infrastructure layer without any compute resources (no VSI, nor OpenShift cluster)
```

Comment:
Reuse from subdirectory readme

```suggestion
- The VPC pattern deploys a simple IBM Cloud VPC infrastructure without any compute resources like VSIs or Red Hat OpenShift clusters
```

Also seems like the order here should be
1. vpc
2. vsi
3. roks
(see comment at line 38 about the table of images)